components/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/model/BusinessOwner.java

```diff
@@ -17,27 +17,24 @@
 */
 package org.wso2.carbon.appmgt.api.model;
 
-import java.util.Map;
+import java.util.List;
 
+/**
+ * Business owner DTO class.
+ */
 public class BusinessOwner {
 
     private int businessOwnerId;
     private String businessOwnerName;
     private String businessOwnerEmail;
     private String businessOwnerDescription;
     private String businessOwnerSite;
-    private Map<String, String> businessOwnerCustomProperties;
-
-
+    private List<BusinessOwnerProperties> businessOwnerPropertiesList;
 
     public BusinessOwner() {
```

Comment:
We do not need default constructor if we don't have any parameterized constructors. The compiler will add it.